### ISSUE TITLE
trim page titles

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wcag#2875]
* [ ] **all** &mdash; to end of all titles, add: 
```
| WCAG | WAI | W3C
```
(like [ACT Rules](https://www.w3.org/WAI/standards-guidelines/act/rules/about/) and others)
* [ ] **Understanding docs** &mdash; Delete "Success Criterion" in "Understanding Success Criterion 1.1.1: Non-text Content" so it's "Understanding 1.1.1: Non-text Content"
* [x] _leave Guideline in "Understanding Guideline 1.1: Text Alternatives"_
* [ ] **Techniques** &mdash; Delete the full technique phrase from title. Just have "Technique" and the ID,
   * ideally with the technology/category written out, e.g.,:
   > Technique General 34 | WCAG | WAI | W3C
   > Technique HTML 56 | WCAG | WAI | W3C
   > Technique Failure 78 | WCAG | WAI | W3C
   * OK if only the letter, e.g.,:
   > Technique G34 | WCAG | WAI | W3C
   > Technique H56 | WCAG | WAI | W3C
   > Technique F78 | WCAG | WAI | W3C

**Comment by mbgower** [comment]
The Understanding pages have undergone some style revamps, I believe since this issue was created.
They now present "Understand SC" in smaller text, with only the short name of the criterion in large text. The same occurs with techniques.

<img width="692" height="163" alt="Image" src="https://github.com/user-attachments/assets/05ffb8bc-14bf-45e9-9cb0-2a86612aafa4" />

This seems to provide a similar outcome to the changes you suggested. If you agree, feel free to close this issue. If you believe your proposal should still be pursued, please provide some clarifying comments.

**Comment by shawna-slh** [comment]
My initial comment is about the ```<title>``` (not the ```<h1>```). :-)

I think it is still relevant.
